static/js/ajax-episode-search.js

```diff
@@ -224,8 +224,6 @@ $(document).ready(function() {
 
         $('.epManualSearch').on('click', function(event) {
             event.preventDefault();
-            var performSearch = '0';
-            var showAllResults = '0';
 
             // @TODO: Omg this disables all the manual snatch icons, when one is clicked
```

Comment:
I'll look into this soon as well. It may need to wait till it's moved to vue though since it's a lot easier that way.